src/components/Form/ProfileForm.tsx

```diff
@@ -80,17 +95,29 @@ function ProfileForm({ callback }: { callback?: () => void }) {
           name: values.name,
           about: values.about,
           skills: values.skills,
-          web3mailPreferences: user.description?.web3mailPreferences,
+          // Don't update this field if User is Collaborator => Needs to be Owner
+          web3mailPreferences:
```

Comment:
we seem to have a conflict here in this form with multiple needs. 
This one aims to update the current logged in user.